rosbag2_transport/test/rosbag2_transport/test_play_timing.cpp

```diff
@@ -29,6 +29,7 @@
 
 #include "rosbag2_transport_test_fixture.hpp"
 
+using namespace std::chrono_literals;  // NOLINT
```

Comment:
Just a tip, but instead of the NOLINT you can also just include only the used operator:

```c++
using std::chrono_literals::operator""s;
```

Reply:
We may need to use other chrono literals in the same file further on. This is a common practice to put `// NOLINT` there.